apps/chrome-devtools/src/services/state.service.ts

```diff
@@ -57,7 +77,7 @@ export class StateService {
     effect(async () => {
       const key = this.activeStateNameStorageKey();
       if (key) {
-        const name = (await chrome.storage.sync.get(key))[key] as string | undefined;
+        const { [key]: name } = (await chrome.storage.sync.get(key)) as Record<string, string | undefined>;
```

Comment:
Does it really work?
![image](https://github.com/user-attachments/assets/6b76c998-6373-4a26-a808-0ed1dd55786f)